templates/base.twig

```diff
@@ -33,12 +24,110 @@
 			</aside>
 		{% endif %}
 	</main>
-
+  </div>
 	{% block footer %}
 	<footer id="footer">
 		{% include 'footer.twig' %}
 	</footer>
 	{{ function('wp_footer') }}
 	{% endblock %}
+
```

Comment:
Also here, seems to be various things lifted from the front end library that aren't necessary for the header. New branch explaining what this code is for would be better, even if's a small one.

Reply:
Yes I will create another branch for header and footer markup implementation and remove it from header block branch [ feature/1265-theme-header ]